.github/workflows/build-and-test.yml

```diff
@@ -69,3 +69,13 @@ jobs:
       run: |
         pushd toolkit
         sudo env "PATH=$PATH" make go-imager go-imagecustomizer go-osmodifier REBUILD_TOOLS=y
+
+  build-docs:
```

Comment:
Action is failing: https://github.com/microsoft/azure-linux-image-tools/actions/runs/12403436233/workflow

I think this should be:

```yaml
build-docs:
  name: go build and validate
  runs-on: ubuntu-latest
  steps:
```